i18n/locale-en.json

```diff
@@ -0,0 +1,223 @@
+{
```

Comment:
The test may not be that important. Is there a fallback mechanism? Ideally this is flexible, but in any case, a key not found in a certain locale should be taken from the base locale (English). See https://www.mediawiki.org/wiki/Manual:MediaWiki_architecture for probably more than you want to know now :-).